src/codegen/build_module.cc

```diff
@@ -85,25 +85,29 @@ namespace target {
 Target llvm() {
   std::unordered_set<std::string> keys({ "llvm", "cpu" });
   std::vector<std::string> options;
-  return Target("llvm", kDLCPU, 512, 1, keys, options, {});
+  return Target("llvm", kDLCPU, 512, 1, keys, options,
+           std::unordered_set<std::basic_string<char>>());
```

Comment:
std::unordered_set<std::string>() instead

Reply:
same for other places

Reply:
Done